mParticle-Tune/MPKitTune.m

```diff
@@ -65,11 +81,11 @@ - (instancetype)initWithConfiguration:(NSDictionary *)configuration startImmedia
     }
 
     _platform = @"ios";
-    MPDevice *device = [[MPDevice alloc] init];
-    _identifierForAdvertiser = device.advertiserId;
-    MPApplication *application = [[MPApplication alloc] init];
+    _identifierForAdvertiser = [self advertiserId];
+    NSDictionary *bundleInfoDictionary = [[NSBundle mainBundle] infoDictionary];
+    NSString *bundleIdentifier = bundleInfoDictionary[@"CFBundleIdentifier"];
```

Comment:
Thanks for asking. Incidentally, this is actually the exact code that was being run from MPApplication already. I checked and the string is getting passed to `NSURLQueryItem#queryItemWithName:value:`, which specifically marks the value as nullable. So I think we should be ok.